sequence/sequences/compose/arith.py

```diff
@@ -0,0 +1,74 @@
+from sequence.core.core import Sequence
+
+import numpy as np
+
+
+class Arith(Sequence):
+    """Sequences combined via arithmetic operation, e.g., addition, multiplication.
+
+    Parameters
+    ----------
+    sequences : List[Sequence]
+        List of sequences to combine.
+    operation : None, str, function, or numpy ufunc, optional, default = None = mean
+        if str, must be one of "mean", "+" (add), "*" (multiply), "max", "min"
```

Comment:
if you take the vector space of all integer sequences, you have that it is close by taking addition, multiplication, min and max (actually is an abstract Riesz space). But it is not close by taking `mean`. If we restrict the class just to consider integer sequence, this can be a problem.

Moreover, what do you mean for `mean`? Is the pointiwise mean? i.e., if `a_n` and `b_n` are two sequences, then the `mean(a_n, b_n)` is the sequence `(a_n + b_n )/ 2` ?

Reply:
yes, pointwise mean. I copied this coding without much thinking from `sktime` distances, where mean is a sensible default.

There is of course also a question about scope here: are you interested only in integer sequences? Seems like an unnecessary restriction.

Reply:
> There is of course also a question about scope here: are you interested only in integer sequences? Seems like an unnecessary restriction.

Actually I'm interested in all kind of sequences but I started with the integer ones which are already very interesting and in some sense easy to work with as you know their image.

The main ideas was to code as much as possible sequences from https://oeis.org

Reply:
> The main ideas was to code as much as possible sequences from https://oeis.org/

Yes, I get that ofc.
Naively, I'm thinking, surely there must be an API, but I cannot find sth with few minutes googling.

Reply:
symbolic resolution would also be neat, e.g., asking, is `OEIS(xyz) + OEIS(abc) = OEIS(def)`, as integer series

Reply:
This is actually a good idea.

I have to admit that, I see the philosophy behind but I don't see how to implement it :-)

From this PR are coming so many good point of inspiration. I'm thinking to open a GitHub-project (or using the one already done) to clarify every point and decide how to proceed :-)

Reply:
thanks, absolutely!

How about meeting on discord (e.g., one of the sktime channels), and discussing a mini-roadmap for sequentium, and the "sets" project?

Reply:
Yes... I will contact you... good idea